content/sensu-go/5.20/reference/assets.md

```diff
@@ -60,6 +60,8 @@ If the downloaded artifact's SHA512 checksum matches the checksum provided by th
 Set the backend or agent's local cache path with the `--cache-dir` flag.
 Disable assets for an agent with the agent `--disable-assets` [configuration flag][30].
 
+Use [token substitution][3] for asset paths on disk to retrieve changes to assets without manually updating your resource commands.
```

Comment:
I'm not sure to understand that bit. Right now the asset path on the filesystem is not done yet (https://github.com/sensu/sensu-go/issues/3662).

Reply:
I'm sorry, I think we thought the whole asset templating thing would be part of 5.20. It sounds like that's incorrect, right?

Reply:
That's right, I don't think the whole asset templating functionalities will be part of 5.20